.ci/install_nemu.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+source /etc/os-release || source /usr/lib/os-release
+
+arch=$("${cidir}"/kata-arch.sh -d)
+
+clone_build_install() {
+	local nemu_repo=https://github.com/intel/nemu.git
+	local nemu_repo_dir="${HOME}/nemu"
+	local nemu_build_dir=
+	local nemu_bin=
+	git clone "${nemu_repo}" "${nemu_repo_dir}"
+	pushd "${nemu_repo_dir}"
+	git fetch origin
+	git checkout experiment/automatic-removal
+
+	case "$arch" in 
+	x86_64)
+		nemu_build_dir="${HOME}/build-${arch}_virt/${arch}_virt-softmmu/"
+		nemu_bin="qemu-system-${arch}_virt"
+		SRCDIR="$PWD" ./tools/build_x86_64_virt.sh
+		;;
+	aarch64)
+		nemu_build_dir="${HOME}/build-${arch}/${arch}-softmmu/"
```

Comment:
We need to keep an eye on https://github.com/kata-containers/documentation/pull/266#discussion_r223599571 as I'd be surprised if the aarch64 binary didn't also include "`virt`".

Reply:
yes, the best would be to test under an arm vm before merging, but unfortunately I don't have one.